docs/Brain-Team/2021-Software/index.md

```diff
@@ -0,0 +1,69 @@
+# Goals
+
+The 2021 Kit software aims to replace the 2015 software, with a launch targeted for SR2021 Kickstart.
```

Comment:
SR2021 is the virtual competition that's just been launched, we're targeting SR2022 afaik.

Reply:
Good point. I had in my head that SR2021 was in 2022 for some reason.

Reply:
b5b4957